recipe/meta.yaml

```
@@ -1,18 +1,19 @@
 {% set name = "fastparquet" %}
-{% set version = "2024.2.0" %}
+{% set version = "2024.11.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/{{ name }}/{{ name }}-{{ version  }}.tar.gz
-  sha256: 81a8f60c51793eb2436b4fdbbf115ff8578a4a457a179240bc08f9d9573d57a4
+  sha256: e3b1fc73fd3e1b70b0de254bae7feb890436cb67e99458b88cb9bd3cc44db419
 
 build:
   number: 0
-  #skip s390x because of lack of cramjam and numba
-  skip: true  # [py<38 or s390x]
+  # skip s390x because of lack of cramjam and numba
+  # skip py>312 because of cramjam suport for 3.12
+  skip: true  # [py<39 or py>312 or s390x]
```

```suggestion
  skip: true  # [py<39 or s390x]
```
Fastparquet does advertise being compatible with 3.13. 
Not including the configuration in abs.yaml is enough to skip the 3.13 build.
Once we get cramjam, this package will be rebuilt via the buildout.